app/views/answers/_answer.html.erb

```diff
@@ -0,0 +1,27 @@
+<p>
+  <%= form_for answer, url: answer_votes_path(answer), method: :post do |f| %>
+    <%= hidden_field_tag 'vote', 'up' %>
+    <%= f.submit '+' %>
+  <% end %>
+  <div>
+    <%= answer.net_votes %>
+    <%= answer.content %>
+    <% if current_user == answer.user %>
+    <%= link_to 'delete', "/questions/#{answer.question.id}/answers/#{answer.id}", method: :delete %>
+    <% end %>
+  </div>
+  <%= form_for answer, url: answer_votes_path(answer), method: :post do |f| %>
+    <%= hidden_field_tag 'vote', 'down' %>
+    <%= f.submit '-' %>
+  <% end %>
+
+  <div>
+    by <%= answer.user.username %>
```

Comment:
Think about adding a delegate on answer for username
`delegate :username, to: user, prefix: false, allow_nil: true` will generate a username method on your answer model.